e2e/commands/start/start_test.go

```diff
@@ -132,4 +130,22 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"repository_authorization\": \"%s\"`, testutil.NormalizePathToAssertInJSON(repoAuthorization))))
 		})
 	})
+
+	When("--output-format and --json-output-file is passed", func() {
```

Comment:
Could add the others output types besides json ? i think it's sonarqube and a invalid type.

Reply:
Added txt and sonoarqube testecases.